csv-pricing.md

```diff
@@ -0,0 +1,118 @@
+
+1. Provide a file path for your CSV pricing data in values.yaml. This path can reference a local PV or an S3 bucket. 
+
+    ```
+    pricingCsv:
+    enabled: true
+    location:
+    provider: "AWS|GCP"
+    region: "us-east-1" 
+    URI: < your-valid-file-URI >
+    csvAccessCredentials: pricing-schema-access-secret
+    ```
+    
+    S3 bucket access is supported with the format `s3://<bucket-name>/<key>`, for example 's3://kc-csv-test/pricing_schema.csv
+`. Otherwise, the CSV file will be read from the cost-analyzer PV, for example:
+
+    ```
+    pricingCsv:
+    enabled: true
+    location:
+    URI: /var/kubecost-csv/custom-pricing.csv
+    location: local
+    extraVolumes:
+    - name: kubecost-csv
+    configMap:
+    name: csv-pricing
+    extraVolumeMounts:
+    - name: kubecost-csv
+    mountPath: /var/kubecost-csv
+    ```
+    
+2. For S3 locations, provide file access. The required permissions are:
+
+    ```
+    {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+           "Resource": [
+                "arn:aws:s3:::<your-bucket-name>/*",
+                "arn:aws:s3:::<your-bucket-name>"
+            ]
+        }
+      ]
+    }
+    ```
+   
+   There are two options for adding the credentials to the Kubecost pod:
+    1. Service key: Create an S3 service key with the permissions above, then add its ID and access key as a K8s secret:
+        1. `kubectl create secret generic pricing-schema-access-secret -n kubecost --from-literal=AWS_ACCESS_KEY_ID=id --from-literal=AWS_SECRET_ACCESS_KEY=key`
+        2. The name of this secret should be the same as csvAccessCredentials in values.yaml above
+    2. [AWS IAM (IRSA) service account annotation](https://docs.aws.amazon.com/eks/latest/userguide/adot-iam.html)
+
+3. Create a CSV in this [format](https://github.com/kubecost/cost-analyzer-helm-chart/blob/gpu-pricing-1.99-rc.1/custom-pricing.csv) (also in the below table). CSV changes are picked up hourly by default. 
+    1. EndTimeStamp: currently unused
+    2. InstanceID: identifier used to match asset
+    3. Region filter match based on topology.kubernetes.io/region 
+    4. AssetClass: node pv, gpu are supported 
+    5. InstanceIDField: field in spec or metadata that will contain the relevant InstanceID. For nodes, often spec.providerID , for pv’s often metadata.name
+    6. InstanceType: optional field to define the asset type, e.g. m5.12xlarge
+    7. MarketPriceHourly: hourly price to charge this asset
+    8. Version: field for schema version, currently unused
+
+    If the node label topology.kubernetes.io/region is present, it must also be in the CSV Region column.
+
+![Pricing table](https://raw.githubusercontent.com/kubecost/docs/main/images/pricing.png)
+
+## GPU pricing:
```

Comment:
There should be an empty line after H* tags and at the beginning of a list. A single line break will satisfy both requirements here.

Reply:
Can you explain this more? You mean a line break after "## GPU pricing" and where the lists start? It seems to look fine in the GH preview.

Reply:
Sure. It's best practices. I use linting in VSCode, so it yells at me when I'm not following best practices.
https://www.markdownguide.org/basic-syntax/#heading-best-practices
https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md
MD022, MD032